package.json

```diff
@@ -35,5 +35,10 @@
 		"node": "22.21.1",
 		"pnpm": "10.23.0"
 	},
+	"pnpm": {
+		"overrides": {
+			"vitest>vite": "catalog:"
```

Comment:
would recommend unconditional `"vite"` selector here, unless you have a need for different versions elsewhere